rllib/models/specs/specs_jax.py

```diff
@@ -0,0 +1,22 @@
+from typing import Tuple, Any, Union
```

Comment:
ignore the JAX stuff for now. Getting JAX to work on ci is coming soon. The jax tests pass locally though. I will turn on all jax tests after the jax dependencies have been added to the ci.